modules/standard/DateTime.chpl

```diff
@@ -643,11 +695,21 @@ module DateTime {
   }
 
   /* Return the offset from UTC */
+  deprecated "'time.utcoffset()' is deprecated. Please use 'time.utcOffset()' instead"
   proc time.utcoffset() {
     if tzinfo.borrow() == nil {
       return new timedelta();
     } else {
-      return tzinfo!.utcoffset(datetime.now());
+      return tzinfo!.utcOffset(datetime.now());
+    }
+  }
+
+  /* Return the offset from UTC */
+  proc time.utcOffset() {
+    if tzinfo.borrow() == nil {
+      return new timedelta();
+    } else {
+      return tzinfo!.utcOffset(datetime.now());
```

Comment:
Can this be called by the deprecated method?

Reply:
Thanks @benharsh! I replaced the bodies of the deprecated methods with calls to their replacements.